assets/data/cards_pt-br.json

```
@@ -202,6 +209,13 @@
       ],
       "description": "É um programa de computador que simula um ser humano em uma conversa por meio de mensagens de texto ou de voz, muito utilizado para automatizar tarefas repetitivas e fornecer suporte e assistência aos usuários. Os chatbots podem ser usados em uma ampla variedade de aplicações, como atendimento ao cliente, assistentes virtuais, entre outros."
     },
+    {
+      "title": "Cidades Inteligentes",
+      "tags": [
+        "Conceito"
+      ],
+      "description": "Cidades Inteligentes (Smart Cities), são localidades cuja infraestrutura dispõe de organização urbana e aparatos digitais voltados à otimização de seus processos. Para tanto, é possível fazer uso de dispositivos tecnológicos de Inteligência Artificial ou Internet das Coisas por exemplo. "
```

O que acha dessa sugestão? Tentei deixar um pouco mais direto e fluido.

```suggestion
      "description": "Cidades Inteligentes (Smart Cities) são localidades que utilizam infraestrutura urbana organizada e tecnologias digitais para otimizar seus processos. Isso inclui o uso de dispositivos com Inteligência Artificial e Internet das Coisas, entre outros."
```